src/popup/components/TransactionDetailsBase.vue

```diff
@@ -196,7 +196,8 @@
           <template #value>
             <TokenAmount
               :amount="+gasPrice"
-              symbol="AE"
+              :protocol="PROTOCOL_AETERNITY"
+              :symbol="AE_SYMBOL"
```

Comment:
This is a "Base" component so in my opinion passing the AE related constants here looks off. Maybe the parent should pass this down as a prop?

Reply:
This component needs refactoring as it has some protocol specific things - I didn't want to tackle this in this PR